helix-core/src/main/java/org/apache/helix/controller/dataproviders/ManagementControllerDataProvider.java

```diff
@@ -19,9 +19,26 @@
  * under the License.
  */
 
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.helix.HelixConstants;
+
+/**
+ * Data provider for controller management mode pipeline.
+ */
 public class ManagementControllerDataProvider extends BaseControllerDataProvider {
-  // TODO: implement this class to only refresh required event types
-  public ManagementControllerDataProvider(String clusterName, String name) {
-    super(clusterName, name);
+  private static final List<HelixConstants.ChangeType> FULL_REFRESH_PROPERTIES =
```

Comment:
NIT: This naming could be confusing? With these change types, it does not require full refresh but these properties refresh, right?